jetty-core/jetty-core-server/src/test/java/org/eclipse/jetty/core/server/HttpConnectionTest.java

```diff
@@ -530,17 +530,21 @@ public void testSimple() throws Exception
     @Test
     public void testEmptyNotPersistent() throws Exception
     {
-        String response = _connector.getResponse("GET /R1?empty=true HTTP/1.0\r\n" +
-            "Host: localhost\r\n" +
-            "\r\n");
+        String response = _connector.getResponse("""
```

Comment:
Is that `\r\n` -> `\r` change expected?

Reply:
yes, text blocks have an implicit \n in them